Vagrantfile

```diff
@@ -24,53 +24,72 @@ Vagrant.configure("2") do |config|
 # Get the VM up-to-date
     sudo apt-get update
 
-# Install opam and things for compiling xapi
-    sudo apt-get install -y opam m4 libxen-dev git
+# Install opam and dependencies for compiling xapi
+    sudo apt-get install -y opam m4 libxen-dev
     opam init -a --compiler=4.02.3 -y
+    eval `opam config env`
     opam remote add xs-opam git://github.com/xapi-project/xs-opam
```

Comment:
Seemed to work fine in my test; it's interesting why it didn't fail though. Maybe it falls back to another protocol if it fails to use git in that remote URL? I'll check tomorrow to be sure.